src/tests/functional/inference_engine/ir_serialization/rt_info_deserialization.cpp

```diff
@@ -472,7 +472,7 @@ TEST_F(RTInfoDeserialization, NodeV11) {
         <layer name="Round" id="1" type="Round" version="opset8">
             <data mode="half_to_even"/>
             <rt_info>
-                <attribute name="fused_names" version="0" value="Round1,Round2"/>
+                <attribute name="fused_names" version="0" value="Round 1  ,  Round 2  "/>
```

Comment:
Can you add some spaces before "Round 1" as well?

Reply:
done